src/Microsoft.ML.Transforms/NAReplaceTransform.cs

```diff
@@ -947,7 +947,7 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColInfo info, string src
         }
     }
 
-    public sealed class NAReplaceEstimator : IEstimator<NAReplaceTransform>
```

Comment:
>NA [](start = 24, length = 2)

The fewer abbreviations the better. `MissingValueReplacingEstimator`.

Note that there are other types that support missing values (namely, strings and keys), so `NaN` is not accurate

Reply:
Strings no longer support missing values FYI...